training_record/views.py

```diff
@@ -79,7 +79,36 @@ def list_records_for_review(self, request):
         '''Return all offCampusRecords for admin'''
         queryset = self.filter_queryset(self.get_queryset()).filter(
             off_campus_event__isnull=False,
-        )
+            ).extra(select={
+                'is_status_submitted':
```

Comment:
I think this can be done in one `.extra()`. with:
```python
high_priority_status = (STATUS_0, STATUS_1, ...)
.select(select={'priority': 'status in {high_priority_status}').order_by('-priority', '-create_time')
```